src/read_lakeparam.c

```diff
@@ -167,6 +167,13 @@ lake_con_struct read_lakeparam(FILE            *lakeparam,
       sprintf(tmpstr, "Lake area fraction (%f) for cell (%d) specified in the lake parameter file must be a fraction between 0 and 1.", temp.Cl[0], soil_con.gridcel);
       nrerror(tmpstr);
     }
+    if(fabs(1-temp.Cl[0]/veg_con[temp.lake_idx].Cv) > 0.01) {
```

Comment:
Can you run this through uncrustify first? The whitespace is inconsistent with what we use in the updated code. See [here](https://github.com/UW-Hydro/VIC/tree/master/tools/code_format) for info. Use the uncrustify_VIC_c.cfg as the confit file.